mne/stats/regression.py

```diff
@@ -244,19 +250,33 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
            considerations. Psychophysiology, 52(2), 169-189.
     """
     if isinstance(solver, string_types):
+        if solver not in ("cholesky", "ridge"):
+            raise ValueError("No such solver: {0}".format(solver))
+
+        if solver == 'ridge':
+            try:
+                from sklearn.linear_model.ridge import ridge_regression
+
+                def solver(X, y):
+                    return ridge_regression(X, y, solver='cholesky', alpha=1.)
```

Comment:
It depends - this is not for prediction after all, you care about the model coefs, where you might prefer to have low bias. The main benefit of this is you can deal with colinear problems better.

Also the main point of this is to make user-supplanted solvers a bit easier.

I guess I can overload the `solver` kwarg to also allow a float, which would become the regularization parameter?